docs/solarsystem/pds/pds.rst

```diff
@@ -0,0 +1,116 @@
+
+``neptune_arcmodel`` is the choice of which ephemeris to assume for Neptune's ring arcs. accepts a float. must be one of 1, 2, or 3 (see https://pds-rings.seti.org/tools/viewer3_nep.shtml for details). default 3. has no effect if planet != 'Neptune'
+
+Outputs
+---------
+``bodytable`` is a `~astropy.table.QTable` containing ephemeris information on the moons in the planetary system. Every column is assigned a unit from `~astropy.units`. We can get a list of all the columns in this table with:
+
+.. code-block:: python 
+
+	>>> print(bodytable.columns) # doctest: +REMOTE_DATA
+	<TableColumns names=('NAIF ID','Body','RA','Dec','RA (deg)','Dec (deg)','dRA','dDec','sub_obs_lon','sub_obs_lat','sub_sun_lon','sub_sun_lat','phase','distance')>
+
+``ringtable`` is a `~astropy.table.QTable` containing ephemeris information on the individual rings in the planetary system. Every column is assigned a unit from `~astropy.units`. We can get a list of all the columns in this table with:
+
+.. code-block:: python
+
+	>>> print(ringtable.columns) # doctest: +REMOTE_DATA
+	<TableColumns names=('ring','pericenter','ascending node')>
+	
+Note that the behavior of ``ringtable`` changes depending on the planet you query. For Uranus and Saturn the table columns are as above. For Jupiter, Mars, and Pluto, there are no individual named rings returned by the Ring Node, so ``ringtable`` returns None; ephemeris for the ring systems of these bodies is still contained in ``systemtable`` as usual. For Neptune, the ring table shows the minimum and maximum longitudes (from the ring plane ascending node) of the five ring arcs according to the orbital evolution assumed by ``neptune_arcmodel``, e.g.:
+
+.. code-block:: python 
+
+	>>> bodytable, ringtable = RingNode.ephemeris(planet='Neptune', epoch='2022-05-24 00:00') # doctest: +REMOTE_DATA
+	>>> print(ringtable) # doctest: +REMOTE_DATA
+    	   ring    min_angle max_angle
+    	              deg       deg   
+    	---------- --------- ---------
+    	   Courage   53.4818   54.4818
+    	   Liberte  44.68181  48.78178
+    	 Egalite A  33.88179  34.88179
+    	 Egalite B   30.0818   33.0818
+    	Fraternite   16.0818  25.68181
+		
+System-wide data are available as metadata in both ``bodytable`` and ``ringtable`` (if ``ringtable`` exists), e.g.:
+
+.. code-block:: python 
+
+	>>> systemtable = bodytable.meta # doctest: +REMOTE_DATA
+	>>> print(systemtable.keys()) # doctest: +REMOTE_DATA
+	dict_keys(['sub_sun_lat', 'sub_sun_lat_min', 'sub_sun_lat_max', 'opening_angle', 'phase_angle', 'sub_sun_lon', 'sub_obs_lon', 'd_sun', 'd_obs', 'light_time', 'epoch'])
+
+
+Acknowledgements
```

Comment:
We don't normally do an acknowledgement section, but start with the info of what the modules uses at the top.